types/secure-random-password/index.d.ts

```diff
@@ -1,20 +1,28 @@
-// Type definitions for secure-random-password 0.1
+// Type definitions for secure-random-password 0.2
 // Project: https://github.com/rackerlabs/secure-random-password
 // Definitions by: Bjørnar Snoksrud <https://github.com/bjornars>
+//                 GaspardFRDev <https://github.com/GaspardFRDev>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export const lower: string;
-export const upper: string;
 export const consonants: string;
-export const vowels: string;
+export const copyableSymbols: string;
 export const digits: string;
-export const symbols: string;
 export const fullSymbols: string;
-export const copyableSymbols: string;
+export const lower: string;
+export const symbols: string;
+export const upper: string;
+export const vowels: string;
+
+export interface RandomPasswordCharactersSet {
+    characters: string;
+    exactly?: number;
```

Comment:
I'm wondering if I can restrict exactly use in order to forbid this case which isn't allowed :
```
randomPassword({  characters: 
  [
    { characters: srp.symbols, exactly: 1 },
  ]
});
```

Reply:
Generally, speaking, it's hard to prevent calls from providing extra properties (e.g. an `{x: number, y:string}` is usable as an `{x:number}`), but you could probably limit it for literals (where unexpected properties are flagged).  Having said that, I'm not sure what you're trying to prevent.